pkg/transport/handshake.go

```diff
@@ -99,7 +96,7 @@ func (hs SettlementHS) Do(ctx context.Context, dc DiscoveryClient, conn network.
 func MakeSettlementHS(init bool) SettlementHS {
 	// initiating logic.
 	initHS := func(ctx context.Context, dc DiscoveryClient, conn network.Conn, sk cipher.SecKey) (err error) {
-		entry := makeEntryFromTpConn(conn, true)
+		entry := makeEntryFromTpConn(conn)
 
 		// TODO(evanlinjin): Probably not needed as this is called in mTp already. Need to double check.
```

Comment:
This is not yet part of the PR but should be. There is a similar defer call a bit further down.

Reply:
We do not have status anymore no I don't think this is needed anymore.

Reply:
@ersonp I agree. Please delete.